test/integration/toolchains/microsoft/test_msbuilddeps.py

```diff
@@ -28,6 +28,57 @@ def test_msbuilddeps_maps_architecture_to_platform(arch, exp_platform):
     assert expected_import in toolchain
 
 
+@pytest.mark.parametrize(
+    "build_type,arch,configuration,exp_platform",
+    [
+        ("Release", "x86", "Release - Test", "Win32"),
+        ("Debug", "x86_64", "Debug - Test", "x64"),
+    ],
+)
+@pytest.mark.parametrize(
+    "config_key,platform_key",
+    [
+        ("GlobalConfiguration", "GlobalPlatform"),
+        (None, None),
```

Comment:
The ``None``, ``None`` case could have been simplified, as this would already be covered by other tests.
But no prob, once it is done, we can keep it!